src/main/webapp/js/user-page-loader.js

```diff
@@ -57,6 +58,19 @@ function setPageTitle() {
            const messageForm = document.getElementById('message-form');
            messageForm.action = '/messages?recipient=' + parameterUsername;
            messageForm.classList.remove('hidden');
+
+           /** Using 34 because @codestudents.com is 17 characters long
+            * and there's at least 2 people in a direct message
+            */
+           if (parameterUsername.length < 34) {
+             const privateOption = document.getElementById('private-option');
```

Comment:
Is this a new Java thing? No type necessary?

Reply:
What happens if 'private-option' isn't in the doc?

Reply:
I basically copied it from previous code that was given to us (const messageForm, right above this). They didn't put a type, so I didn't either and I didn't receive any errors. Also, nothing would happen if it wasn't in the doc, it's like css files, you can have style specifications for a class, but if the class isn't in the file, nothing happens.